<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeliveryBoy</class>
 <widget class="QWidget" name="DeliveryBoy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="DeliveryBoy_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1001</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="DeliveryBoy_login">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <widget class="QPushButton" name="DeliveryBoy_signup_button_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>340</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#DeliveryBoy_signup_button{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(24, 24, 27, 219), stop:1 rgba(0, 0, 0, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#DeliveryBoy_signup_button:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(86, 86, 86, 219), stop:1 rgba(180, 180, 181, 226));
}
QPushButton#DeliveryBoy_signup_button:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>LogIn</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeliveryBoy_Heading_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LogIn</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeliveryBoy_login_Input_Password">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeliveryBoy_login_Intput_UserName">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackToMenu_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeliveryBoy_signup_button_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>390</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#DeliveryBoy_signup_button{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(24, 24, 27, 219), stop:1 rgba(0, 0, 0, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#DeliveryBoy_signup_button:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(86, 86, 86, 219), stop:1 rgba(180, 180, 181, 226));
}
QPushButton#DeliveryBoy_signup_button:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="seller_menu">
    <widget class="QWidget" name="left_side" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(22, 25, 29);</string>
     </property>
     <widget class="QWidget" name="Menu" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Menu_Button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Menu_Button{	
	border-radius:7px;
	background-color: rgb(246, 218, 232);
}

</string>
       </property>
       <property name="text">
        <string>  Menu</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/home.svg</normaloff>../icons/home.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Others" native="true">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>109</y>
        <width>261</width>
        <height>571</height>
       </rect>
      </property>
      <widget class="QPushButton" name="All_item_list">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Add_product{	
	border-radius:7px;
background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>  All Items List</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/add-product.png</normaloff>../icons/add-product.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="Items_to_deliver">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Remove_product{	
	border-radius:7px;
	background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>  Items To Deliver</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/truck.svg</normaloff>../icons/truck.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="Update_password">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Update_password{	
	border-radius: 7px;
	background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>  Update Password</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/settings.svg</normaloff>../icons/settings.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="Items_to_deliver_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Remove_product{	
	border-radius:7px;
	background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>  Update Status</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/changes.png</normaloff>../icons/changes.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="SignUp_page">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <widget class="QLineEdit" name="DeliveryBoy_signup_Intput_UserName">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>240</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeliveryBoy_signup_button">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>580</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#DeliveryBoy_signup_button{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(24, 24, 27, 219), stop:1 rgba(0, 0, 0, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#DeliveryBoy_signup_button:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(86, 86, 86, 219), stop:1 rgba(180, 180, 181, 226));
}
QPushButton#DeliveryBoy_signup_button:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeliveryBoy_signup_Input_password">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>520</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeliveryBoy_Heading_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delivery Boy SignUp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeliveryBoy_signup_Intput_email">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeliveryBoy_signup_Intput_address">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>380</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeliveryBoy_signu_Intput_contact">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>450</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackToMenu_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>650</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_error">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
